### PR TITLE
[Integration test] Add support for DD_TAGS and DD_EXTRA_TAGS

### PR DESCRIPTION
**Pull request by maxday**
- Add integration tests for DD_TAGS and DD_EXTRA_TAGS for both metrics and tags
- Trigger Github-Action on push on files related to integration tests to make review easier (and not to wait for the cron job to see whether or not tests are successful)

### CODE REVIEW

File: integration_tests/recorder-extension/src/index.js [hunk 1]
```diff
@@ -87,8 +87,15 @@ async function next(extensionId) {
     app.post('/v1/input', async (req, res) => {
         if(JSON.stringify(req.body) !== '{}') { // to avoid printing empty logs due to the connectivity test
             for(let i = 0; i < req.body.length; ++i) {
+                //sort tags to avoid flakky tests
```
Comment (nhinsch) [inline integration_tests/recorder-extension/src/index.js:90]: flakky -> flaky

File: integration_tests/recorder-extension/src/index.js [hunk 2]
```diff
@@ -87,8 +87,15 @@ async function next(extensionId) {
...
+                const originalTags = req.body[i].ddtags;
+                const sortedTags = originalTags.split(",");
+                sortedTags.sort();
+                // reset tags once sorted
+                req.body[i].ddtags = sortedTags.join(",")
                 const logString = JSON.stringify(req.body[i]);
                 if(logString.indexOf("[sketch]") === -1 && logString.indexOf("[log]") === -1) { // avoid inception
```
Comment (nhinsch) [inline integration_tests/recorder-extension/src/index.js:97]: What do you mean by "avoid inception" here? I would make this comment a bit more specific.

**Review by nhinsch — APPROVED** [review]
Looks great! Hooray for integration tests!

File: integration_tests/recorder-extension/src/index.js (hunk 1 above)
Comment (maxday) [reply integration_tests/recorder-extension/src/index.js]: ✅

File: integration_tests/recorder-extension/src/index.js (hunk 2 above)
Comment (maxday) [reply integration_tests/recorder-extension/src/index.js]: ✅

**Pull request merged**